src/v2/Apps/Artwork/Components/ArtworkSidebar/index.tsx

```diff
@@ -37,13 +38,17 @@ export const ArtworkSidebar: React.FC<ArtworkSidebarProps> = ({
   )
 
   // If we have info about the lot end time (cascading), use that.
-  const { sale, saleArtwork } = artwork
+  const { sale, saleArtwork, is_sold, is_in_auction } = artwork
   const endAt = saleArtwork?.endAt
   const startAt = sale?.startAt
   const { hasEnded } = useTimer(endAt!, startAt!)
 
+  const shouldSkipCreateAlert =
```

Comment:
**suggestion**: I wonder if this might be misinterpreted in the future because we still have a create alert CTA under these conditions, it's just rendered somewhere else. Maybe something like `shouldHideDetailsCreateAlertCTA` might be more clear/verbose?

Reply:
Yeah, I like this name 👍